packages/capnpc-ts/src/generators.ts

```diff
@@ -325,11 +326,18 @@ export function generateStructFieldMethods(
 
       }
 
+      const listClassIdentifier = ts.createIdentifier(listClass);
+
       adopt = true;
       disown = true;
       get = ts.createCall(
-        ts.createPropertyAccess(
-          THIS, '_getList'), __, [offsetLiteral, ts.createIdentifier(listClass)]);
+        ts.createPropertyAccess(THIS, '_getList'),
+        __,
+        [offsetLiteral, listClassIdentifier]);
+      init = ts.createCall(
```

Comment:
I'll leave it up to you, but you might want to pull in [this commit](https://github.com/jdiaz5513/capnp-ts/commit/32df29647c3f9dc181b9b62b9d4eaef308fd0292) instead.